internal/konnect/sdk/sdk.go

```diff
@@ -0,0 +1,20 @@
+package sdk
+
+import (
+	sdkkonnectgo "github.com/Kong/sdk-konnect-go"
+	sdkkonnectcomp "github.com/Kong/sdk-konnect-go/models/components"
+)
+
+// SDK returns a new SDK instance.
+func SDK(token string, opts ...sdkkonnectgo.SDKOption) *sdkkonnectgo.SDK {
```

Comment:
Maybe `sdk.New` will make more sense than `sdk.SDK`, WDYT?
```suggestion
func New(token string, opts ...sdkkonnectgo.SDKOption) *sdkkonnectgo.SDK {
```

Reply:
bc5d7a6773e5f10c86c2e2716716a599c65b0a55